cxAstScan/test/success_cancel_before_scan.ts

```diff
@@ -0,0 +1,19 @@
+import tmrm = require('azure-pipelines-task-lib/mock-run');
```

Comment:
Is this file equal to cxAstScan/test/success_cancel.ts?
If so, could we use the same file?